recipes/atomsk/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "atomsk" %}
+{% set version = "0.10.1" %}
+{% set sha256 = "28dac7c83a7d0b76b773eb37d921300fef8120faa829d47eae133d8e2889b22b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pierrehirel/{{ name }}/archive/Beta-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - gcc
```

Comment:
Could you please change this part to the new conda build 3? like
```
- {{ compiler('c') }}
```

Reply:
That’s what I tried in 
https://github.com/conda-forge/staged-recipes/pull/5920/commits/bde69fcb57dbc06432fe63a09f1235e23fd0a2df But somehow it did not work.

Reply:
Could you please send me the link of the CI with the errors?

Reply:
https://circleci.com/gh/conda-forge/staged-recipes/27636